templates/env.rc

```diff
@@ -114,8 +114,8 @@ fi
 export OPENSTACK_CLOUD_PROVIDER_CONF_B64="$(cat ${CAPO_CLOUD_PROVIDER_CONF_TMP} | base64 --wrap=0)"
 
 # Build OPENSTACK_CLOUD_CACERT_B64
-CAPO_OPENSTACK_CLOUD_CACERT=""
+CAPO_OPENSTACK_CLOUD_CACERT_B64=""
```

Comment:
I'm not sure if that leads to errors when CAcert is empty (as in my case). Maybe rather like this?
```suggestion
CAPO_OPENSTACK_CLOUD_CACERT_B64=$(echo "" | base64 --wrap=0)
```

Reply:
yes, fixed